Sources/Stagehand/Driver/InteractiveDriver.swift

```diff
@@ -0,0 +1,246 @@
+// Created by Nick Entin on 3/3/25.
+
+import Foundation
+import UIKit
+
+@MainActor
+internal final class InteractiveDriver {
```

Comment:
should this conform to `Driver`?

Reply:
also, you're marking as `internal` explicitly here, but we're not doing that on methods. intentional?

Reply:
Nah, marking internal was a copy from elsewhere cause I did that a lot in this framework originally.

And no, shouldn't conform to Driver. I actually tried that originally, but Driver is really for AnimationInstance, and the interactive API is different enough it doesn't make sense to share the API.